core/vm/gas_table.go

```diff
@@ -17,9 +17,9 @@
 package vm
 
 import (
-	"github.com/ShyftNetwork/go-empyrean/common"
-	"github.com/ShyftNetwork/go-empyrean/common/math"
-	"github.com/ShyftNetwork/go-empyrean/params"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/common/math"
+	"github.com/ethereum/go-ethereum/params"
```

Comment:
I'm concerned with these lines, we should not be pulling from main ethereum

Reply:
Updated all files imports